packages/indicators/README.md

```diff
@@ -36,18 +38,22 @@ const result = simpleMovingAverage(data, { periods: 3, field: 'close' });
 
 ### Calculate an array of values from an array of numbers:
 
-```
+```js
 const data = [1, 2, 3, ...];
 const result = simpleMovingAverageArray(data, { periods: 10 });
 ```
 
+<br />
+
 ## Exponential Moving Average (EMA)
 
 The 12- and 26-day EMAs are the most popular short-term averages, and they are used to create indicators like the moving average convergence divergence (MACD) and the percentage price oscillator (PPO). In general, the 50- and 200-day EMAs are used as signals of long-term trends.
 
 -   [Moving Average @ Wikipedia](https://en.wikipedia.org/wiki/Moving_average)
 -   [EMA @ Investopedia](http://www.investopedia.com/terms/e/ema.asp)
 
+<br />
+
```

Comment:
do you need this two line breaks?
i think `<br />` is enough and you can remove the second one
we have some of these in this file. please check them all